packages/@webex/webex-core/src/lib/services/services.js

```diff
@@ -52,6 +52,7 @@ const Services = WebexPlugin.extend({
 
   props: {
     validateDomains: ['boolean', false, true],
+    initFailed: ['boolean', false, true],
```

Comment:
if I'm reading this correctly, the `false, true` bit means that it's not required and has a default `true` - is that right? so we say that init failed before init is attempted? That doesn't feel right....

Reply:
yeah it wasn't right. I changed `initFailed: 'boolean'` to add the default sequence after writing the check boolean test as it was returning undefined.. and got the default the wrong way round. Tests now cover this.